scripts/apps/authoring/compare-versions/directives/CompareVersionsArticleDirective.js

```diff
@@ -59,6 +72,109 @@ class LinkFunction {
 
         this.scope.isLocked = this.lock.isLocked(item);
     }
+
+    /**
+     * @ngdoc method
+     * @name sdCompareVersionsArticle#setVersionsDifference
+     * @param {Object} item - current item version
+     * @param {Object} oldItem - old item to compare with
+     * @description Changes 'headline', 'abstract', 'body_footer', 'body_html', 'byline' from item
+     * in order to highlight the differences from oldItem
+     */
+    setVersionsDifference(item, oldItem) {
```

Comment:
imo would be better to return a diff and not modify the item, otherwise you have to keep a copy somewhere